tomviz/modules/ModuleVolume.cxx

```diff
@@ -287,8 +306,7 @@ void ModuleVolume::onTransferModeChanged(const int mode)
 
 vtkSmartPointer<vtkDataObject> ModuleVolume::getDataToExport()
 {
-  vtkTrivialProducer* trv = this->dataSource()->producer();
-  return trv->GetOutputDataObject(0);
+  return m_imageData.GetPointer();
```

Comment:
We don't need GetPointer()/Get() anymore - can use the implicit cast to pointer in the vtkNew/vtkSmartPointer objects.